clean_project/project/search_engine.py

```diff
@@ -21,11 +23,50 @@ def search(self, search_word):
         sslmode = load_config_file(self.connect_path).get('sslmode')
         connection = psycopg2.connect(host=host, port=port, sslmode=sslmode,
                                       database=database, user=user, password=password)
+        words = tuple(search_word.split(" "))
```

Comment:
ok so if you pass a list it doesn't convert automatically the list to be printed in the statement with round brackets?